domains/jogar.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Henzogabriel954",
+           "email": "gabrielhenzo316@gmail.com",
+           "discord": "1066758638032662530"
+        },
+    
+        "record": {
+            "A": ["144.22.224.131"]
```

Comment:
Can't reach the website, if this is the correct record, could you provide a preview of your website?